### PR TITLE
 Luxeradio-portfolio-landing-section

### PR DESCRIPTION
**Pull request by cp-priyanka-g**

### CODE REVIEW

File: vue-frontend/src/components/partials/ScreenHeaderV2.vue [hunk 1]
```diff
@@ -29,6 +29,9 @@
                 class="v2-normal-3-text"
                 :class="[
                   navbar.className,
+                  currentRoutePath == '/portfolio/luxeradio'
+                    ? 'show-links'
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/partials/ScreenHeaderV2.vue:33]: As we are hiding link, name should be relatable

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/partials/ScreenHeaderV2.vue [hunk 2]
```diff
@@ -18,7 +18,7 @@
             />
           </div>
         </a>
-        <div class="navbar-collapse">
+        <div class="navbar-collapse show-header">
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/partials/ScreenHeaderV2.vue:21]: What is `show-header` usage?

**Comment by cp-shruti-s** [comment]
landing_background.webp image is needed ??

**Comment by cp-priyanka-g** [comment]
> 



> landing_background.webp image is needed ??

it is used in landing section of home-new page

**Comment by cp-sumi-k** [comment]
Add text animation as per cuberto

**Review by cp-sumi-k — APPROVED** [review]

**Review by cp-shruti-s — APPROVED** [review]

**Pull request merged**